src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="search_input_big form"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="small"
+      action="#"
+    >
+      <input
+        class="search_input_small form"
+        type="text"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
The input element in the second form is missing the `data-qa` attribute, which is present in the first form's input element. Ensure consistency by adding a `data-qa` attribute to this input as well, similar to the first form.